### DISCUSSION TITLE
Please help me with the problems encountered in developing desktop programs

### DISCUSSION THREAD

**Discussion by penghu0** [unoplatform/uno#6839] [Q&A]
## Please help me with the problems encountered in developing desktop programs
1：How to fix the width and height of the startup form？
2：How to disable window maximization？

Nuget Package Version(s):

Affected platform(s):
- [ ] macOS
- [X ] Skia
  - [ ] WPF
  - [ X] GTK (Linux)
  - [ ] Tizen

IDE:
-  Visual Studio 2019 (version: )
-  Visual Studio for Mac (version: )

**Comment by jeromelaban** [comment]
Window sizing [has been merged](https://github.com/unoplatform/uno/pull/6604) very recently, and you can use `ApplicationView.PreferredLaunchViewSize` to change it.

For the Window Maximization configuration, it's not supported yet because there's API to do so in WinRT/WinUI. We may align to this feature instead one it progresses: https://github.com/microsoft/microsoft-ui-xaml/issues/2564. Still, depending on the platform you may use native APIs to achieve the same effect.
